tests/test_uploads.py

```diff
@@ -38,6 +38,16 @@ def test_upload_multiple_fields_with_the_same_name(self, httpbin):
         assert r.count(FILE_CONTENT) in [3, 4]
         assert r.count('Content-Type: text/plain') == 2
 
+    def test_upload_custom_content_type(self, httpbin):
+        r = http('--form', '--verbose', 'POST', httpbin.url + '/post',
+                 'test-file@%s;type=image/vnd.microsoft.icon' % FILE_PATH_ARG)
+        assert HTTP_OK in r
+        # Content type is stripped from the filename
+        assert 'Content-Disposition: form-data; name="test-file";' \
+               ' filename="%s"' % os.path.basename(FILE_PATH) in r
```

Comment:
You can use f-strings!

Reply:
You can tell Python is not my main language, can you? :o)
Will fix

Reply:
Haha. But this is the only thing that gave you away :)